lib/assets/traits/device_item_icon.dart

```diff
@@ -12,6 +13,8 @@ class DeviceItemIcon {
       return LockIcon(determiningTrait.state.value);
     } else if (determiningTrait is ThermostatTrait) {
       return buildThermostatIcon(determiningTrait.state.value);
+    } else if (determiningTrait is PowerTrait) {
+      return buildPowerIcon(determiningTrait.state.value);
```

Comment:
Maybe we can do switch case here. The build thermostatIcon or powerIcon are just returning the object so maybe we can just have PowerItemIcon here and maybe similar for ThermostatIcon

Reply:
Fixed.